components/brave_wallet/browser/asset_ratio_response_parser_unittest.cc

```diff
@@ -7,32 +7,40 @@
 #include <utility>
 #include <vector>
 
-#include "base/i18n/time_formatting.h"
+#include "base/json/json_reader.h"
 #include "base/strings/string_util.h"
+#include "base/values.h"
 #include "brave/components/brave_wallet/browser/asset_ratio_response_parser.h"
 #include "brave/components/brave_wallet/browser/brave_wallet_utils.h"
 #include "brave/components/brave_wallet/common/brave_wallet.mojom.h"
 #include "testing/gtest/include/gtest/gtest.h"
 
 namespace brave_wallet {
 
+namespace {
+base::Value ToValue(const std::string& json) {
+  return base::JSONReader::Read(json).value_or(base::Value());
```

Comment:
perhaps some utils from `values_test_utils.h`could be handier, e.g. `ParseJson` or `ParseJsonDict`

Reply:
TIL values_test_util exists :)